docs/fsharp-cheatsheet.md

```diff
@@ -75,6 +92,12 @@ Other common examples are `F` or `f` for 32-bit floating-point numbers, `M` or `
     // [fsi:val d : decimal = 0.7833M]
     // [fsi:val bi : System.Numerics.BigInteger = 9999]
 
+*Primes* (or a tick `'` at the end of a label name) are idiomatic to functional languages and are included in F#. They are part of the identifier's name and simply indicate to the developer a variation of an existing value or function. For example:
```

Comment:
I can be convinced otherwise, but I didn't think it was a good fit for this. It all seems a little too obvious.

Reply:
I think it's ok to have in an appendix - it definitely is something JS or CS people won't have seen so saying "this thing is not special, it's just a diff char that F# allows in the name" is of value IMO

and it can live alongside triple backticks - people never shut up about it in intros but its only used in some places. saying "used for test names, probably should not be used willy nilly" is helpfuy;

BUT, in here, it's distracting to the point being made, so a footnote ref would be the ideal representation